pixeltable/config.py

```diff
@@ -32,7 +32,11 @@ def __init__(self) -> None:
 
         self.__home = Path(os.environ.get('PIXELTABLE_HOME', str(Path.home() / '.pixeltable')))
         if self.__home.exists() and not self.__home.is_dir():
-            raise RuntimeError(f'{self.__home} is not a directory')
+            raise RuntimeError(
+                "ERROR initializing Pixeltable: The Pixeltable home path ('{{self.__home}}') must be a directory, "
```

Comment:
What do the double brackets do? There is no f-specifier here so I think this will result in the exact string `{{self.__home}}` being shown to the user, which has no meaning to a user